tests/_ide_helper.php

```diff
@@ -0,0 +1,19577 @@
+<?php
```

Comment:
Was this file commited on purpose?

It seems `generateAndGetIdeHelperFileContent` generates it all the time anyway?

Reply:
yes，it's a test temp file. it will provides its content to test.